lasagne/layers/conv.py

```diff
@@ -48,7 +49,7 @@ def __init__(self, incoming, num_filters, filter_length, stride=1,
             self.nonlinearity = nonlinearity
 
         self.num_filters = num_filters
-        self.filter_length = filter_length
+        self.filter_size = filter_size
```

Comment:
Wohoo, another name discussion! ;) `filter_shape` would also be consistent with Theano, but we don't necessarily need that. As a counterexample, `scipy.ndimage.median_filter` has a `size` keyword that can be a scalar or tuple. And I agree that `pool_shape=3` looks weird, as if we would enumerate possible shapes (1: square, 2: triangle, 3: circle).

We could just make it a rule that things that *must* be tuples are `shape`, and things that can also be scalars are `size`. If we follow that everywhere, this would still count as consistent.

Regarding the original comment, I thought about that, too, but I think `as_tuple(filter_size, 1)` would make the rest of the code a bit harder to read. It could be helpful for refactoring the code, though (there's a lot of duplication between the convolution classes).